src/runtime/cuda.cpp

```diff
@@ -395,11 +408,11 @@ WEAK CUresult create_cuda_context(void *user_context, CUcontext *ctx) {
 
         // threads per core is a function of the compute capability
         int threads_per_core = (cc_major == 1 ? 8 :
-                                cc_major == 2 ? (cc_minor == 0 ? 32 : 48) :
-                                cc_major == 3 ? 192 :
-                                cc_major == 5 ? 128 :
-                                cc_major == 6 ? (cc_minor == 0 ? 64 : 128) :
-                                cc_major == 7 ? 64 : 0);
+                                                cc_major == 2 ? (cc_minor == 0 ? 32 : 48) :
```

Comment:
runaway indentaton

Reply:
this should be a switch statement instead, anyway